tutorials-dp0-3/portal-dp0-3-5.rst

```diff
@@ -211,14 +223,13 @@ The query will return all SSSource observation entries for the list of 110 SSObj
     :alt: A screenshot of the fully populated "Object ID Search" section of the UI.
 **A screenshot of the fully populated "Object ID Search" section of the UI.**
```

Comment:
In all cases for all figures, the caption should be indented (see the rst file for portal tutorial 01) so that it properly appears as a caption associated with the figure.

Reply:
Added indentations to the source (but rendered output looks the same to me; so not sure this is what you referred to. Please let me a different edit is needed).